### PR TITLE
Fix typo: seperated -> separated

### PR DESCRIPTION
**Pull request by justincampbell**
Also put a dash between comma-separated

### CODE REVIEW

File: samples/client/petstore/android/default/src/main/java/io/swagger/client/api/PetApi.java [hunk 1]
```diff
@@ -263,7 +263,7 @@ public void  deletePet (Long petId, String apiKey) throws ApiException {
   
   /**
    * Finds Pets by tags
-   * Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+   * Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/android/default/src/main/java/io/swagger/client/api/PetApi.java]: Another `Muliple`.

File: samples/client/petstore/android/volley/src/main/java/io/swagger/client/api/PetApi.java [hunk 2]
```diff
@@ -590,7 +590,7 @@ public void onErrorResponse(VolleyError error) {
   
   /**
   * Finds Pets by tags
-  * Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+  * Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/android/volley/src/main/java/io/swagger/client/api/PetApi.java]: Another `Muliple`.

File: samples/client/petstore/android/volley/src/main/java/io/swagger/client/api/PetApi.java [hunk 3]
```diff
@@ -656,7 +656,7 @@ public void onErrorResponse(VolleyError error) {
 
       /**
    * Finds Pets by tags
-   * Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+   * Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/android/volley/src/main/java/io/swagger/client/api/PetApi.java]: Another `Muliple`.

File: samples/client/petstore/clojure/src/swagger_petstore/api/pet.clj [hunk 4]
```diff
@@ -90,7 +90,7 @@
 
 (defn find-pets-by-tags-with-http-info
   "Finds Pets by tags
-  Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing."
+  Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing."
```
Comment (samrocketman) [inline samples/client/petstore/clojure/src/swagger_petstore/api/pet.clj]: Another `Muliple`.

File: samples/client/petstore/clojure/src/swagger_petstore/api/pet.clj [hunk 5]
```diff
@@ -104,7 +104,7 @@
 
 (defn find-pets-by-tags
   "Finds Pets by tags
-  Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing."
+  Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing."
```
Comment (samrocketman) [inline samples/client/petstore/clojure/src/swagger_petstore/api/pet.clj]: Another `Muliple`.

File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs [hunk 6]
```diff
@@ -111,7 +111,7 @@ public interface IPetApi
         /// Finds Pets by tags
         /// </summary>
         /// <remarks>
-        /// Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+        /// Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs]: Another `Muliple`.

File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs [hunk 7]
```diff
@@ -122,7 +122,7 @@ public interface IPetApi
...
```
Comment (samrocketman) [inline samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs]: Another `Muliple`.

File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs [hunk 8]
```diff
@@ -367,7 +367,7 @@ public interface IPetApi
...
```
Comment (samrocketman) [inline samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs]: Another `Muliple`.

File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs [hunk 9]
```diff
@@ -378,7 +378,7 @@ public interface IPetApi
...
```
Comment (samrocketman) [inline samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs]: Another `Muliple`.

File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs [hunk 10]
```diff
@@ -1316,7 +1316,7 @@ public List<Pet> FindPetsByStatus (List<string> status = null)
         }
         
         /// <summary>
-        /// Finds Pets by tags Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+        /// Finds Pets by tags Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs]: Another `Muliple`.

File: samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs [hunk 11]
```diff
@@ -1412,7 +1412,7 @@ public List<Pet> FindPetsByTags (List<string> tags = null)
         }
 
         /// <summary>
-        /// Finds Pets by tags Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+        /// Finds Pets by tags Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/csharp/SwaggerClientTest/Lib/SwaggerClient/src/main/csharp/IO/Swagger/Api/PetApi.cs]: Another `Muliple`.

File: samples/client/petstore/dart/lib/api/pet_api.dart [hunk 12]
```diff
@@ -155,7 +155,7 @@ class PetApi {
   
   /// Finds Pets by tags
   ///
-  /// Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+  /// Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/dart/lib/api/pet_api.dart]: Another `Muliple`.

File: samples/client/petstore/go/swagger/PetApi.go [hunk 13]
```diff
@@ -272,7 +272,7 @@ func (a PetApi) FindPetsByStatus (status []string) ([]Pet, error) {
 }
 /**
  * Finds Pets by tags
- * Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+ * Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/go/swagger/PetApi.go]: Another `Muliple`.

File: samples/client/petstore/java/default/src/main/java/io/swagger/client/api/PetApi.java [hunk 14]
```diff
@@ -212,7 +212,7 @@ public void deletePet(Long petId, String apiKey) throws ApiException {
   
   /**
    * Finds Pets by tags
-   * Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+   * Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/java/default/src/main/java/io/swagger/client/api/PetApi.java]: Another `Muliple`.

File: samples/client/petstore/java/feign/src/main/java/io/swagger/client/api/PetApi.java [hunk 15]
```diff
@@ -75,7 +75,7 @@
   
   /**
    * Finds Pets by tags
-   * Muliple tags can be provided with comma seperated strings. Use tag1, tag2, tag3 for testing.
+   * Muliple tags can be provided with comma-separated strings. Use tag1, tag2, tag3 for testing.
```
Comment (samrocketman) [inline samples/client/petstore/java/feign/src/main/java/io/swagger/client/api/PetApi.java]: Another `Muliple`.

**Comment by samrocketman** [comment]
Hahaha perhaps just:...
```
grep -rl 'Muliple' | xargs -I{} sed -i 's/Muliple/Multiple/' {}
```
:smile:

**Comment by justincampbell** [comment]
@samrocketman Thanks! Fixed those.

**Comment by ePaul** [comment]
Just as a note: You don't need to fix all those generated files by hand, just generate them again.

**Comment by wing328** [comment]
@justincampbell thanks for the PR to fix the typo.

I've filed #2542 to fix the typo in petstore.yaml (in which I need to restore from petstore server).

I'll do a similar fix to petstore.json later as the spec also needs a clean up to remove some fake endpoints (for testing).

**Pull request closed without merging**